docs/rules/ordered-list-marker-sequence.md

```diff
@@ -16,21 +16,22 @@ since: "v0.12.0"
 
 ## 📖 Rule Details
 
-This rule enforces that ordered list markers in Markdown use strictly sequential numbers. If the numbering skips, decreases, or starts with an unexpected value, this rule will report an error. The rule can automatically format and correct list numbers to ensure consistency throughout your document.
+This rule enforces consistent numbering in ordered list markers. By default, it requires strictly sequential numbers (1, 2, 3, ...). Alternatively, you can configure it to enforce that all items use `1.` as the marker.
+
+Using `1.` for all items simplifies maintenance, makes diffs cleaner, and reduces the chance of renumbering errors when reordering or inserting new list items. All major Markdown parsers render the list with proper sequential numbers regardless of the actual marker values.
```

Comment:
Please also mention what happens if you use sequential numbers.
Both options should be convenient for users, as the rules will format a consistent sequence without the user having to insert them in order.